addons/website/views/website_visitor_views.xml

```diff
@@ -321,7 +321,7 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">website.track</field>
         <field name="view_mode">tree</field>
-        <field name="context">{'search_default_type_url':1}</field>
+        <field name="context">{'search_default_type_url':1, 'create':0, 'edit':0, 'copy':0}</field>
```

Comment:
```suggestion
        <field name="context">{'search_default_type_url': 1, 'create': False, 'edit': False, 'copy': False}</field>
```
(yes also fix the existing 'typo' while editing this line)

Reply:
> (yes also fix the existing 'typo' while editing this line)

Which ?

Reply:
The missing space after `'search_default_type_url'` :+1: